spec/providers/coreIntegration/environment.integration.src.js

```diff
@@ -32,4 +32,26 @@ module.exports = function (setup) {
       done();
     });
   });
+
+  it("exposes dynamic require functionality", function (done) {
+    moduleEnv.testRequire(true).then(function (ret) {
+      expect(ret).toEqual(true);
+      done();
+    }, function (ret) {
+      expect(ret).toEqual(true);
+      expect(true).toEqual(false);
```

Comment:
You just mean that this block should never be reached right? Do we need 2 assertions?